functional/platform/nspawn.go

```diff
@@ -384,14 +384,14 @@ func (nc *nspawnCluster) CreateMember() (m Member, err error) {
 	return nc.createMember(id)
 }
 
-func newMachineID() string {
+func (nc *nspawnCluster) NewMachineID() string {
```

Comment:
Why did you turn it into a method on `nc`, if it doesn't need `nc` for anything?...